
The alx driver is to only support the AR8161 and AR8162
devices. The older devices are supported through atl1c.

--- a/drivers/net/ethernet/atheros/alx/alx_main.c
+++ b/drivers/net/ethernet/atheros/alx/alx_main.c
@@ -33,12 +33,6 @@ static const char alx_drv_description[]
 #define ALX_ETHER_DEVICE(device_id) {\
 	PCI_DEVICE(ALX_VENDOR_ID, device_id)}
 static DEFINE_PCI_DEVICE_TABLE(alx_pci_tbl) = {
-	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8131),
-	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8132),
-	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8151_V1),
-	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8151_V2),
-	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8152_V1),
-	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8152_V2),
 	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8161),
 	ALX_ETHER_DEVICE(ALX_DEV_ID_AR8162),
 	{0,}
